src/probability/process/stopping.lean

```diff
@@ -959,10 +958,50 @@ lemma stopped_value_eq' [preorder ι] [locally_finite_order_bot ι] [add_comm_mo
   stopped_value u τ = ∑ i in finset.Iic N, set.indicator {ω | τ ω = i} (u i) :=
 stopped_value_eq_of_mem_finset (λ ω, finset.mem_Iic.mpr (hbdd ω))
 
+lemma stopped_process_eq_of_mem_finset [linear_order ι] [add_comm_monoid E]
+  {s : finset ι} (n : ι) (hbdd : ∀ ω, τ ω < n → τ ω ∈ s) :
+  stopped_process u τ n =
+  set.indicator {a | n ≤ τ a} (u n) +
+    ∑ i in set.to_finset (↑s ∩ set.Iio n), set.indicator {ω | τ ω = i} (u i) :=
```

Comment:
Would it be easier if we avoid the coercion by phrasing `set.to_finset (↑s ∩ set.Iio n)` as `s.filter (\lam x, x < n)`?

Reply:
Thanks! That's indeed better